src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -22,6 +26,23 @@ class OppgaveController(val oppgaveService: OppgaveService, val saksbehandlerSer
         private val logger = getLogger(javaClass.enclosingClass)
     }
 
+    @ApiOperation(value = "Hent oppgaver", notes = "Henter alle oppgaver som saksbehandler har tilgang til.")
+    @GetMapping("/ansatte/{navIdent}/ikketildelteoppgaver", produces = ["application/json"])
+    fun getOppgaver(
+        @ApiParam(value = "NavIdent til en ansatt")
+        @PathVariable navIdent: String,
+        @ApiParam(value = "Klage eller anke")
```

Comment:
OpenApi har støtte for enums, kan man angi det også med disse annotasjonene?

Reply:
Mener jeg brukt enums før, uten at jeg husker detaljene.

Reply:
Har dog ikke brukt OpenAPI-spec før.